scripts/piped-work/get_gui_structure.py

```diff
@@ -65,7 +65,6 @@ def getStructure() :
     #do( 'Help: CommandName=Help' )
     #do( 'Help: CommandName=SetPreference' )
     #do( 'SetPreference: PrefName=GUI/Theme PrefValue=light' )
-    #do( 'Screenshot: CaptureMode=menus' )
```

Comment:
Wow, I never once in my life looked in this subdirectory of the source tree before!

Did you verify that things still run?

What remains in here can serve as examples for how to use the script pipe.

Reply:
I'm trying to run the old Perl script.  What I didn't know is that first you must start Audacity and have an audio track, not an empty project.  I get frequent error dialogs, and now the new export dialog messes up the macros by pausing for user input.

So anyway making this all run again seems out of scope for now.  But tell @dozzzzer and @chinakov to study these examples if they haven't.  Maybe writing Python programs driving the script pipe for regression tests would be easier than writing macros using Audacity's own user interface.

Reply:
I see `pipe_test.py` too, and there is still mention of screenshot in a comment in it.